tools/project_gen/project_gen/project.py

```diff
@@ -59,7 +60,15 @@ def create_project(
         If True then a completely new project is setup,
         If False then only the results are computed (the results
             directory has to be empty though)
-    use_mp: bool
+    erf_dir: Path, optional
+        Path to the ERF base directory, expected to contain
+         - NZBCK2015_Chch50yearsAftershock_OpenSHA_modType4.txt
+         - NZ_DSmodel_2015.txt
+         - NZ_FLTmodel_2010.txt
```

Comment:
Based on the docstring, do we need the following items inside the `FLT_ERF_MAPPING`?
- NZBCK2015_Chch50yearsAftershock_OpenSHA_modType4
- NZ_DSmodel_2015

Reply:
Nope those are for DS and we currently don't support using any other ones.